src/scripts/clipperUI/componentBase.ts

```diff
@@ -115,10 +126,23 @@ export abstract class ComponentBase<TState, TProps> {
 		};
 	}
 
+	private static focusOnButton(ariaForArrowKeyNav: string, nextPosInSet: number) {
+		const buttons = document.querySelectorAll("a[data-ariaForArrowKeyNav=" + ariaForArrowKeyNav + "]");
```

Comment:
The attribute name should probably be a constant somewhere, so that if we change it, we don't have to change the name everywhere in all the components